### ISSUE TITLE
svd-accordion doesn't have role "tablist" attribute

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#829]
See the https://surveyjs.answerdesk.io/ticket/details/t4333/survey-creator-survey-logic-accessibility-issue-ensure-elements-with-an-aria-role-that thread for more details
